src/components/CustomInputs/DictInput.vue

```diff
@@ -58,9 +64,10 @@ export default {
     },
     value() {
       const dict = {}
-      this.keys
-        .filter(k => k !== null)
-        .map((k, i) => (dict[k] = this.values[i]))
+      const keys = this.includeCheckbox
+        ? this.keys.filter(k => this.paramsToAdd.includes(k))
+        : this.keys
+      keys.filter(k => k !== null).map((k, i) => (dict[k] = this.values[i]))
```

Comment:
Index here doesn't match up with the correct value.  If I only add one param it takes the value from position 0 in the values array.